starlite/middleware/authentication.py

```diff
@@ -1,14 +1,16 @@
 import re
 from abc import ABC, abstractmethod
-from typing import TYPE_CHECKING, Any, List, Optional, Pattern, Union
+from typing import TYPE_CHECKING, Any, List, Optional, Pattern, Set, Union
 
 from pydantic import BaseConfig, BaseModel
 
 from starlite.connection import ASGIConnection
 from starlite.enums import ScopeType
-from starlite.middleware.util import should_bypass_middleware
+from starlite.middleware.utils import should_bypass_middleware
```

Comment:
Maybe we should update CSRF middleware as well in this PR. It also relies on exclude opts to skip protection on certain routes.

I think @provinzkraut simply forgot to uppdate it in #630 when he introduced should_bypass_middleware function

Reply:
good point, but it should be done in a followup

Reply:
Yup, I did!